caikit/config/config.py

```diff
@@ -62,8 +62,9 @@ def configure(
             with overrides for your usage.
         config_dict (Optional[Dict]): Config overrides in dictionary form
 
-    Returns: None
-        This only sets the config object that is returned by `caikit.get_config()`
+    Returns:
+			None: This only sets the config object that is returned by `caikit.get_config()`
```

Comment:
indents look pretty excessive here :D

Reply:
They are! Have this as a set value which I shouldn't.